src/sections/header.liquid

```diff
@@ -0,0 +1,581 @@
+
+{%- if app -%}
+  {%- capture element_app_desktop -%}
+    <div class="{{- class -}}__app">
+      {%- render 'app', block: app, section: section, size: "desktop" -%}
+    </div>
+  {%- endcapture -%}
+{%- endif -%}
+
+{% comment %} CSS variables start {% endcomment %}
+{%- capture variables -%}
+  {%- case padding_bottom_desktop -%}
+    {%- when 'md' -%}
+      --padding-bottom-desktop: 16px;
+    {%- when 'lg' -%}
+      --padding-bottom-desktop: 20px;
+  {%- endcase -%}
+
+  {%- case padding_bottom_mobile -%}
+    {%- when 'md' -%}
+      --padding-bottom-mobile: 16px;
+    {%- when 'lg' -%}
+      --padding-bottom-mobile: 20px;
+  {%- endcase -%}
+
+  {%- case padding_top_desktop -%}
+    {%- when 'md' -%}
+      --padding-top-desktop: 16px;
+    {%- when 'lg' -%}
+      --padding-top-desktop: 20px;
+  {%- endcase -%}
+
+  {%- case padding_top_mobile -%}
+    {%- when 'md' -%}
+      --padding-top-mobile: 16px;
+    {%- when 'lg' -%}
+      --padding-top-mobile: 20px;
+  {%- endcase -%}
+
+  {%- if menu_opener_position == 'left' -%}
+    --menu-indent-mobile: 0 20px 0 0;
+    --order-app-mobile: 3;
+    --order-button-mobile: 6;
+    --order-cart-mobile: 5;
+    --order-logo-mobile: 2;
+    --order-menu-mobile: 1;
+    --order-search-mobile: 4;
+  {%- else -%}
+    --menu-indent-mobile: 0 0 0 8px;
+  {%- endif -%}
+
+  {%- if menu_alignment_vertical == 'top' -%}
+    --logo-growth-desktop: 0;
+    --menu-growth-desktop: 1;
+    --menu-border-width-desktop: 0;
+    --menu-bottom-border-top-width-desktop: 0;
+    --menu-bottom-padding-top-desktop: 0;
+    --menu-bottom-width-desktop: auto;
+    --menu-margin-x-desktop: auto;
+    --menu-indent-desktop: 0 20px;
+    --order-app-desktop: 3;
+    --order-button-desktop: 6;
+    --order-cart-desktop: 5;
+    --order-logo-desktop: 1;
+    --order-menu-desktop: 2;
+    --order-search-desktop: 4;
+  {%- else -%}
+    --menu-bottom-border-top-width-desktop: 100vw;
+    {%- if header_outline == "bordered" -%}
+      --menu-bottom-margin-desktop: {% if padding_bottom_desktop == 'lg' %}20px{% elsif padding_bottom_desktop == 'md' %}16px{% else %}12px{% endif %};
+      --menu-bottom-border-top-width-desktop: calc(100% + var(--menu-bottom-margin-desktop) * 2);
+    {%- endif -%}
+    --menu-bottom-padding-top-desktop: {% if padding_bottom_desktop == 'lg' %}20px{% elsif padding_bottom_desktop == 'md' %}16px{% else %}12px{% endif %} 0 0;
+    --menu-margin-x-desktop: -22px;
+    --menu-indent-desktop: {% if padding_bottom_desktop == 'lg' %}20px{% elsif padding_bottom_desktop == 'md' %}16px{% else %}12px{% endif %} 0 0;
+  {%- endif -%}
+
+  {%- if menu_alignment_horizontal == 'left' -%}
+    --menu-horizontal-position-desktop: flex-start;
+  {%- elsif menu_alignment_horizontal == 'right' -%}
+    --menu-horizontal-position-desktop: flex-end;
+  {%- endif -%}
+{%- endcapture -%}
+{% comment %} CSS variables end {% endcomment %}
+
+{%- if show_mini_cart -%}
+  {%- capture element_mini_cart -%}
```

Comment:
Why do we have to capture the mini cart in a variable first? Can we just write this code directly on line 225?